river/tree/_utils/gradient_hessian.py

```diff
@@ -0,0 +1,130 @@
+import copy
+
+from river import stats
+
+
+class GradHess:
+    """ The most basic inner structure of the Streaming Gradient Trees. """
```

Comment:
This description is not very informative.

Reply:
I expanded it a little bit in commit [940122d](https://github.com/online-ml/river/pull/458/commits/940122dfb81b6e62d7f43a2e51cfb113b90d899d). What do you think?